config/dagu/scripts/git-maintenance.py

```
@@ -0,0 +1,259 @@
+        text=True,
+        timeout=600,
+    )
+    output = result.stdout + result.stderr
+    return result.returncode == 0, output.strip()
+
+
+def run_cleanup(repo: Path) -> tuple[bool, str] | None:
+    cleanup = repo / "scripts" / "cleanup"
+    if not (cleanup.exists() and os.access(cleanup, os.X_OK)):
+        return None
+    result = subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)